### PR TITLE
Melt exome priors ignorecontigs

### PR DESCRIPTION
**Pull request by ViktorHy**
# Description
MELT configuration has been reworked to further facilitate better recalls. Firstly, known variants that melt previously have called have been added to a priors-list to help guide MELT callling. This is recommended by MELT developers. Second, exome flag was added to signify that input data is not WGS. Third, cov flag was removed as it was incorrectly used as standard deviation of insert size, instead by removing this flag the default values are used, which are recommended by developers (it is what was used during training of MELT).

To confirm the changed all previously called insert-elements for 25 samples were run using the new configs and resulted in 100% recall.

## Type of change
- [ ] Major change 
- [x] Minor change
- [ ] Patch
- [ ] Documentation

# Checklist:

- [x] My code follows the [style guidelines for NF pipelines at CMD](http://mtlucmds1.lund.skane.se/wiki/doku.php?id=nextflow&s[]=nextflow#code_style_at_cmd)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings (Keep an eye on `.nextflow.log` !)
- [x] I have updated the CHANGELOG
- [x] The latest commit in the master branch is tagged
      with the correct version number

## Minor change
- [x] Stub run completes without errors or new warnings
- [x] `onco` run finishes without any new warnings/errors and the results can 
       be loaded into scout
- [x] `wgs` single run finishes without any new warnings/errors and the results 
      can be loaded into scout
- [x] `wgs` trio run finishes without any new warnings/errors and the results 
       can be loaded into scout
- [x] At least one other person has reviewed and approved my code
- [x] I have made corresponding changes to the documentation

# Instructions for the reviewers
Verification report has been written and signed. But please test code implementation, stub run and one oncov2 and one oncov1 into scout

## Expected outcome

There should not be any noticeable difference but for MELT recallability.

# Review

## Review performed by:
- Reviewer 1: Name1, (\@Github\_handle1)  
- Reviewer 2: Name1,  (\@Github\_handle2)
- ...
    
## Testing performed by:
- Name, (\@Github\_handle)
- ...

# Post-merge
- [ ] The `master` branch has been tagged with the new version number.

### CODE REVIEW

**Comment by Jakob37** [comment]
I'll start checking!

"I have updated CHANGELOG" is checked, but I see nothing in the diff. Are all edits pushed?

File: configs/nextflow.hopper.config [hunk 1]
```diff
@@ -155,7 +155,7 @@ profiles {
     params.panelhs = "sentieon driver -r ${params.genome_file} -t ${params.cpu_all} -i "
     params.panelhs2 = " --algo HsMetricAlgo --targets_list ${params.intervals} --baits_list ${params.intervals} hs_metrics.txt"
     params.bed_melt = "${params.refpath}/bed/MELT/Hg38.genes.bed"
-    params.mei_list = "${params.refpath}/annotation_dbs/MELT/mei_list.txt"
+    params.mei_list = "${params.refpath}/annotation_dbs/MELT/mei_list_withpriors"
```
Comment (Jakob37) [inline configs/nextflow.hopper.config:158]: A file suffix is nice for ease of understanding. Not sure whether it is a tsv or txt, but could add a txt suffix as before?

**Comment by Jakob37** [comment]
Onco v1 and v2 ran through without errors.

Remaining things to check:

- [ ] Compare onco v1 outputs to previous runs
- [ ] Compare onco v2 outputs to previous runs
- [ ] Check stub-runs

**Review by Jakob37 — APPROVED** [review]
Looks good! Runs completed without errors and could be loaded into Scout. Stub run went well as well.

I compared the MELT outputs for the two onco samples to how they looked in my runs prior to this PR.

In the onco v1 I the number of found Alu repeats increased from 160 to 518. The number found within the intersect went from 1 to 2.

In the onco v2 the number of found Alu repeats increased from 18 to 153 (none found in the intersects).

All the previously found were still there, so it seems to increase recall without dropping out anything previously found.

**Comment by Jakob37** [comment]
Make sure to update the CHANGE log though!

File: configs/nextflow.hopper.config (hunk 1 above)
Comment (ViktorHy) [reply configs/nextflow.hopper.config:158]: fine

**Comment by ViktorHy** [comment]
Yeah never pushed the CHANGELOG.md changes

**Pull request merged**